content/minfraud/api-documentation.mdx

```diff
@@ -76,6 +76,33 @@ determine whether it has been used in previous fraudulent transactions. The
 device information is factored into the scores returned through the minFraud
 service.
 
+## Timeout Errors
+
+If you get timeout errors, first confirm if we experienced a service disruption
+on our [status page](https://status.maxmind.com/) or on
+[Pingdom](http://stats.pingdom.com/datuqzybqg5z).
+
+If there has not been a service disruption, we may be able to help you diagnose
+your connection issues. Please use our [mm-network-analyzer
+program](https://github.com/maxmind/mm-network-analyzer), which is designed to
+aid in diagnosing networking issues. It creates a zip file containing various
+network checks like traceroute, DNS, etc. You can download the latest version,
+run it, and send the resulting zip file to our support team for review.
+
+Please note, the mm-network-analyzer is not compatible with Windows, but you can
```

Comment:
```suggestion
Please note, `mm-network-analyzer` is not compatible with Windows, but you can
```

Reply:
i added the backticks around mm-network-analyzer throughout the copy and made your requested changes